docs/design/software-design-specification.adoc

```diff
@@ -543,7 +544,7 @@ For all repositories in scope, the artifacts in the corresponding groups in Nexu
 
 | SDS-EXT-26
 | NodeJS 16 for UBI 8
-| 1
+| 16
```

Comment:
I think this was correct being `1`. `1` is the tag used by RedHat ... The `16` is already in the image name ...